database/migrations/2025_03_31_124158_support_model_name_on_site.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        // Add position column on measurement table.
+        echo "Adding 'processormodelname' column on 'siteinformation' table" . PHP_EOL;
```

Comment:
No need to add an extra `echo` here given that the migration is very simple.  Artisan already prints a message when the migration is being run.